app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,10 @@
     package="com.nervousfish.nervousfish"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-permission android:name="android.permission.BLUETOOTH" />
+    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
```

Comment:
I feel very uncomfortable about this, especially since many of our users will be very unhappy with having to allow our app to get their location...

But apparently [you're right](https://developer.android.com/about/versions/marshmallow/android-6.0-changes.html#behavior-hardware-id) (link to the [StackOverflow answer](http://stackoverflow.com/a/34967068/4132379)) 🙁